streaming-binlog/src/main/java/org/openmrs/analytics/model/EventConfiguration.java

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package org.openmrs.analytics.model;
+
+import java.util.LinkedHashMap;
+
+public class EventConfiguration {
```

Comment:
Have you considered using [Lombok](https://projectlombok.org/features/GetterSetter)?

Reply:
Nice suggestion let me use this instead.